docs/cli-add-view.rst

```diff
@@ -5,18 +5,24 @@ This tool will add a new view.
 
 .. code-block:: shell-session
 
-    python ocdskingfisher-views-cli add-view [viewname] [collection_id1] "[note]"
-    python ocdskingfisher-views-cli add-view [viewname] [collection_id1],[collection_id2] "[note]"
+    python ocdskingfisher-views-cli add-view [collection_id1] "[note]"
+    python ocdskingfisher-views-cli add-view [collection_id1],[collection_id2] "[note]"
 
 For example,
 
 .. code-block:: shell-session
 
-    python ocdskingfisher-views-cli add-view viewname 1,3 "Created by Fred."
+    python ocdskingfisher-views-cli add-view 1,3 "Created by Fred."
 
-The view name must be something that is a valid Postgres schema name. We suggest using numbers, characters and _ only.
+This will create a schema called `view_data_collection_1_3`.  You can pass a single collection ID, or multiple collection ID's separated by commas.
 
-You can pass a single collection ID, or multiple collection ID's separated by commas.
+If you select more than 5 collection or if you want to choose your own view name you can pass a --name argument.  The view name must be something that is a valid Postgres schema name. We suggest using numbers, characters and _ only.
```

Comment:
"If you select more than 5 collections" <- s

Reply:
done